automem/api/memory.py

```
@@ -681,4 +682,221 @@ def associate() -> Any:
+                {"memories": validated},
+            )
+        except Exception:
+            logger.exception("Batch graph write failed")
+            abort(500, description="Failed to store memories in FalkorDB")
+
+        # 4. Batch Qdrant upsert
+        qdrant_client = get_qdrant_client()
+        qdrant_status = "unconfigured"
+        if qdrant_client is not None:
+            points = []
+            for v, emb in zip(validated, embeddings):
```

Ruff (0.15.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)